lightkurve/targetpixelfile.py

```diff
@@ -51,13 +51,23 @@ class KeplerTargetPixelFile(TargetPixelFile):
         http://archive.stsci.edu/kepler/manuals/archive_manual.pdf
     """
 
+
     def __init__(self, path, quality_bitmask=KeplerQualityFlags.DEFAULT_BITMASK,
                  **kwargs):
         self.path = path
         self.hdu = fits.open(self.path, **kwargs)
+        self._test_file()
         self.quality_bitmask = quality_bitmask
         self.quality_mask = self._quality_mask(quality_bitmask)
 
+    def _test_file(self, keys=['FLUX', 'QUALITY']):
+        """Tests if the file has FLUX and QUALITY data.
```

Comment:
For clarity, I'd change the docstring into `Raises a ValueError exception if `self.hdu` does not appear like a Target Pixel File.`.